OsmAnd/src/net/osmand/plus/views/MapTileLayer.java

```diff
@@ -77,7 +93,25 @@ public void setAlpha(int alpha) {
 		super.setAlpha(alpha);
 		if (paintBitmap != null) {
 			paintBitmap.setAlpha(alpha);
-		}
+		}//*
+		if (view != null) {
+			final MapRendererView mapRenderer = view.getMapRenderer();
+			if (mapRenderer != null && !oldRender) {
+				OsmandSettings st = view.getApplication().getSettings();
+				if (!Algorithms.objectEquals(st.MAP_TRANSPARENCY.get(), cachedMapTransparency)) {
+					cachedMapTransparency = st.MAP_TRANSPARENCY.get();
+					MapLayerConfiguration mapLayerConfiguration = new MapLayerConfiguration();
+					mapLayerConfiguration.setOpacityFactor(((float) cachedMapTransparency) / 255.0f);
+					mapRenderer.setMapLayerConfiguration(0, mapLayerConfiguration);
+				}
+				if (!Algorithms.objectEquals(st.MAP_OVERLAY_TRANSPARENCY.get(), cachedOverlayTransparency)) {
```

Comment:
This class should not have any reference to MAP_OVERLAY_TRANSPARENCY & MAP_TRANSPARENCY, it's not needed cause Canvas method doesn't use it.